libs/ui-web/src/lib/components/ui-web-image-host/ui-web-image-config.ts

```diff
@@ -3,7 +3,7 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class UiWebImageConfig {
   src: string;
-  width?: number;
-  height?: number;
+  width?: number | string;
+  height?: number | string;
```

Comment:
Since we're using `[style.width|height]` this should be typed as `string` instead of `number | string`. HTML attributes are always `string`.